config/routes.rb

```diff
@@ -65,6 +65,9 @@
   post "/coronavirus-form/business-details" => "coronavirus_form/business_details#submit"
 
   # Question 12
+  get "/coronavirus-form/business-details" => "coronavirus_form/business_details#show"
```

Comment:
This route and the one below look to be the same as those on line 64 and 65?

Reply:
Good spot!  Came from a rebase that didn't quite go to plan.